tests/data_context/test_data_context.py

```diff
@@ -279,6 +280,36 @@ def test_save_expectation_suite(data_context_parameterized_expectation_suite):
     assert expectation_suite.expectations == expectation_suite_saved.expectations
 
 
+@pytest.mark.unit
+def test_save_expectation_suite_include_rendered_content(
+    data_context_parameterized_expectation_suite,
+):
```

Comment:
Is this truly a unit test? Not 100% sure what moving pieces are here but just trying to narrow our definitions.

Reply:
I agree, this marker was a mistake.

Reply:
✅